api/src/main/java/io/jsonwebtoken/JwtParserBuilder.java

```diff
@@ -298,8 +298,8 @@
     JwtParserBuilder deserializeJsonWith(Deserializer<Map<String,?>> deserializer);
 
     /**
-     * Returns a {@link JwtParser} created from the configuration from this JwtParserBuilder.
-     * @return a JwtParser created from the configuration from this JwtParserBuilder.
+     * Returns a immutable/thread-safe {@link JwtParser} created from the configuration from this JwtParserBuilder.
+     * @return a immutable/thread-safe JwtParser created from the configuration from this JwtParserBuilder.
```

Comment:
Technically `an immutable/thread-safe`..., but I'll still approve 😉

Reply:
nice catch, i'll fix that before merging